tests/integration/zome_call.rs

```diff
@@ -0,0 +1,87 @@
+use base64::prelude::*;
+use fixt::prelude::*;
+use holochain::fixt::DnaHashFixturator;
+use holochain_http_gateway::{
+    config::{AllowedFns, Configuration},
+    tracing::initialize_tracing_subscriber,
+};
+use reqwest::StatusCode;
+
+use crate::TestApp;
+
+#[tokio::test]
+async fn zome_call_uses_correct_route_parameters() {
+    initialize_tracing_subscriber("info");
```

Comment:
It's fine to have a default level for the tracing subscriber but as the person running the test, I'd like to be able to change the log level without changing the code

Reply:
Removed the param entirely, will revert to default if `RUST_LOG` is not set in the environment